pkg/remotewrite/config.go

```diff
@@ -25,32 +25,32 @@ var defaultTrendStats = []string{"p(99)"}
 
 type Config struct {
 	// URL contains the absolute URL for the Write endpoint where to flush the time series.
-	URL null.String `json:"url" envconfig:"K6_PROMETHEUS_REMOTE_URL"`
+	URL null.String `json:"url" envconfig:"K6_PROMETHEUS_RW_REMOTE_URL"`
```

Comment:
hmm this seems like duplication, how about just `K6_PROMETHEUS_RW_URL`? or `K6_PROMETHEUS_RW_SERVER_URL` / `K6_PROMETHEUS_RW_SERVER_ADDR`

Reply:
I'm fine with `K6_PROMETHEUS_RW_URL` and `K6_PROMETHEUS_RW_SERVER_URL`. `K6_PROMETHEUS_RW_SERVER_ADDR` isn't good because often the implementations require a specific path, so it has to be passed here also.

For example this is the default value:

https://github.com/grafana/xk6-output-prometheus-remote/blob/203e3b6105a3f5c679dd5a880fdbb8523cbe046f/pkg/remotewrite/config.go#L18

Reply:
`K6_PROMETHEUS_RW_SERVER_URL` has my vote then, though I am also fine with just `K6_PROMETHEUS_RW_URL`